test/system/helpers.bash

```diff
@@ -137,11 +137,16 @@ function run_podman() {
         '?')             expected_rc=  ; shift;;  # ignore exit code
     esac
 
+    local BIN=$PODMAN
+    if [[ ! -z "$COVERAGE" ]]; then
+        BIN="$PODMAN -test.coverprofile=coverprofile.system.$(random_string 20) COVERAGE"
+    fi
+
     # stdout is only emitted upon error; this echo is to help a debugger
-    echo "$_LOG_PROMPT $PODMAN $*"
+    echo "$_LOG_PROMPT $BIN $@"
```

Comment:
It's kind of complicated, and I'll explain one day over a beer, but please leave that as dollar-asterisk

Reply:
Yes and yes :-) Thanks for checking!